### PR TITLE
fix(enviroment): solve dev env variables

### LINKED ISSUE #627: Stating test variables
Some hapi staging env variables are empty

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### GH Issue

Resolves #627

### Steps to test

1. Make run
1. Make fresh with bad endpoint
1. Check hapi docker-compose variables are same than staging-envarioment

### CODE REVIEW

File: .github/workflows/push-master-environment.yaml [hunk 1]
```diff
@@ -82,14 +82,17 @@ jobs:
...
           # hapi
-          POSTGRES_HOST: ${{ secrets.POSTGRES_HOST }}
-          POSTGRES_PORT: ${{ secrets.POSTGRES_PORT }}
-          POSTGRES_SCHEMA: ${{ secrets.POSTGRES_SCHEMA }}
-          HAPI_EOS_API_ENDPOINT: https://api.eosio.cr
           HAPI_SERVER_ADDRESS: 0.0.0.0
           HAPI_SERVER_PORT: 9090
-          HAPI_PROXY_CONTRACT: ${{ secrets.HAPI_PROXY_CONTRACT }}
-          HAPI_RATING_CONTRACT: ${{ secrets.HAPI_RATING_CONTRACT }}
+          HAPI_POSTGRES_USER: ${{ secrets.HAPI_POSTGRES_USER }}
```
Comment (leisterfrancisco) [inline .github/workflows/push-master-environment.yaml:87]: @xavier506 please confirm me that variables values are correct

File: .github/workflows/push-staging-environment.yaml [hunk 2]
```diff
@@ -82,14 +82,17 @@ jobs:
           POSTGRES_PASSWORD: ${{ secrets.POSTGRES_PASSWORD }}
           POSTGRES_USER: ${{ secrets.POSTGRES_USER }}
           # hapi
-          POSTGRES_HOST: ${{ secrets.POSTGRES_HOST }}
-          POSTGRES_PORT: ${{ secrets.POSTGRES_PORT }}
-          POSTGRES_SCHEMA: ${{ secrets.POSTGRES_SCHEMA }}
-          HAPI_EOS_API_ENDPOINT: https://jungle3.cryptolions.io
           HAPI_SERVER_ADDRESS: 0.0.0.0
```
Comment (leisterfrancisco) [inline .github/workflows/push-staging-environment.yaml:85]: @xavier506 same here

File: hapi/src/libs/sync-bps.js [hunk 3]
```diff
@@ -13,10 +13,19 @@ const getBlockProducersData = async () => {
     verbose: false
   })
   
-  const { rows: producers } = await eos.getProducers({
-    json: true,
-    limit: 1000
-  })
+  let producers
```
Comment (leisterfrancisco) [inline hapi/src/libs/sync-bps.js:16]: @xavier506 here I made some correction due to some bad url connection

**Review by xavier506 — APPROVED** [review]
LGTM - will merge and adjust env vars as required

**Pull request merged**